spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -302,39 +307,39 @@
 
   let(:creative_commons_attribution_v3_url) { "http://creativecommons.org/licenses/by/3.0/us/" }
   let(:expected_creative_commons_attribution_v3_url) do
-    {
+    [{
```

Comment:
I'm not sure what happened with this. I was pretty aggressively testing so maybe something got overwritten. Either way, this fixes the breaking specs.